bingo/bingo-elastic/python/bingo_elastic/model/record.py

```diff
@@ -108,6 +109,12 @@ class IndigoRecord:
     elastic_response = WithElasticResponse()
     record_id: Optional[str] = None
     error_handler: Optional[Callable[[object, BaseException], None]] = None
+    # Sort for page cursor
+    _sort: list
```

Comment:
Seems that `IndigoRecord` not the right place for storing sort cursor. This is representation of IndigoObject. Should we move `_sort` to other place?

Reply:
Personal opinion i think since elastic_response = WithElasticResponse() is already in IndigoRecord, _sort can fit here as well. Since _sort is under the context of elastic response. It is used as the cursor of the current elastic search under the elastic_response.

I don't mind an alternative storage location though. What would you suggest?